cirq/google/vis/heatmap.py

```diff
@@ -0,0 +1,250 @@
+"""Convenience functions for visualizations.
+
+The main design goal is to make the interface easy to use for most common use
+cases, yet powerful enough for customizations.
+
+The public entry points are:
+
+* class Heatmap:  a heatmap of qubit metrics.
+"""
+
+from typing import (Any, Dict, List, Mapping, Optional, SupportsFloat, Tuple,
+                    Union)
+
+import numpy as np
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import pandas as pd
+from matplotlib import collections as mpl_collections
+from mpl_toolkits import axes_grid1
+
+QubitCoordinate = Tuple[int, int]
```

Comment:
Should we also support GridQubit, so type `Union[Tuple[int, int], GridQubit]`?

Reply:
Done. The `test_cells_positions()` test is updated to test both types.